lib/kmer_hash.cc

```diff
@@ -93,5 +94,66 @@ std::string _revhash(HashIntoType hash, WordLength k)
     return s;
 }
 
+std::string _revcomp(const std::string& kmer)
+{
+    std::string out = kmer;
+    size_t ksize = out.size();
+
+    for (int i=0; i < ksize; ++i) {
+        char complement;
+
+        switch(kmer[i]) {
+        case 'A':
+            complement = 'T';
+            break;
+        case 'C':
+            complement = 'G';
+            break;
+        case 'G':
+            complement = 'C';
+            break;
+        case 'T':
+            complement = 'A';
+            break;
+        default:
+            throw khmer::khmer_exception("Invalid base in read");
+            break;
+        }
+        out[ksize - i - 1] = complement;
+    }
+    return out;
+}
+
+HashIntoType _hash_murmur(const std::string& kmer)
+{
+    HashIntoType h = 0;
+    HashIntoType r = 0;
+
+    return khmer::_hash_murmur(kmer, h, r);
+}
+
+HashIntoType _hash_murmur(const std::string& kmer,
+                          HashIntoType& h, HashIntoType& r)
+{
+    HashIntoType out[2];
+    uint32_t seed = 0;
+    MurmurHash3_x64_128((void *)kmer.c_str(), kmer.size(), seed, &out);
```

Comment:
I can cast to int, but it will only silence the warning (the [original MurmurHash][1] code uses "const int").

[1]: https://github.com/ged-lab/khmer/pull/257/files#diff-71364610b0b275f8566888e96600dff2R33